### PR TITLE
allow configurable role fields

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by jquense** [comment]
ah so I went with a tuple selfishly so i can support one or the field as we migrate to piecemeal to the newer auth0 login flows.

**Comment by itajaja** [comment]
I have to say the tuple makes for a pretty weird API

**Comment by jquense** [comment]
totes. couldn't think of a nicer way of handling one or another fields at once :/

**Comment by itajaja** [comment]
you could just subclass it in our internal utils package for the time being?

**Comment by jquense** [comment]
sure true

File: flask_resty_tenants/authorization.py [hunk 1]
```diff
@@ -48,7 +49,7 @@ def get_tenant_id(self, model_or_item):
 
     def get_role_data(self):
         try:
-            role_data = self.get_request_credentials()['app_metadata']
+            role_data = self.get_request_credentials()[self.role_field]
```
Comment (itajaja) [inline flask_resty_tenants/authorization.py:53]: `role_fields`

**Review by itajaja — APPROVED** [review]

File: flask_resty_tenants/authorization.py [hunk 2]
```diff
@@ -21,6 +21,7 @@ class TenantAuthorization(HasCredentialsAuthorizationBase):
     global_tenant = '*'
     tenant_id_type = UUID
     tenant_id_field = 'tenant_id'
+    role_field = 'app_metadata'
```
Comment (taion) [inline flask_resty_tenants/authorization.py]: i would put this in a newline separated group above the tenant things

**Review by taion — APPROVED** [review]
tiny nit

**Pull request merged**